tools/pip/MANIFEST.in

```diff
@@ -29,4 +29,5 @@ recursive-include mxnet *_LICENSE
 recursive-include mxnet *.h
 recursive-include mxnet *.hpp
 recursive-include mxnet *.cuh
+recursive-include mxnet *.cpp
```

Comment:
does *.cpp include *.cc files?
https://github.com/apache/incubator-mxnet/blob/master/src/lib_api.cc

Reply:
actually no, let me include cc as well.

Reply:
cool, were you able to package the wheel and unzip it and confirm the lib_api.cc was inside?